include/mbedtls/timing.h

```diff
@@ -27,6 +27,11 @@
 
 #include <stdint.h>
 
+#if ( defined(__ARMCC_VERSION) || defined(_MSC_VER) ) && \
+    !defined(inline) && !defined(__cplusplus)
+#define inline __inline
+#endif
+
```

Comment:
Note: it looks like you did not make the getter inline after all, so this is probably not required. OTOH, it clearly doesn't hurt. (No change required, just thinking out loud.)

Reply:
It was initially inline, but this just became too messy to support with the `MBEDTLS_HAVE_TIME` changes. I will remove this anyway.